Foundation/NSGeometry.swift

```diff
@@ -91,6 +95,61 @@ public func ==(lhs: CGPoint, rhs: CGPoint) -> Bool {
     return lhs.x == rhs.x && lhs.y == rhs.y
 }
 
+extension CGPoint: NSSpecialValueCoding {
+    init(bytes: UnsafePointer<Void>) {
+        let buffer = UnsafePointer<CGFloat>(bytes)
+
+        self.x = buffer.memory
+        self.y = buffer.advancedBy(1).memory
+    }
+    
+    public init?(coder aDecoder: NSCoder) {
```

Comment:
NSSpecialValueCoding is an internal protocol however it is true that the coding methods are public. How about I remove NSSpecialValueCoding inheriting from NSSecureCoding and make all these methods internal? It's not an API change if the methods are not exported, right?